x-pack/solutions/security/packages/data_table/components/data_table/helpers.test.tsx

```diff
@@ -178,7 +177,7 @@ describe('helpers', () => {
   });
 
   describe('addBuildingBlockStyle', () => {
-    const THEME = { eui: euiThemeVars, darkMode: false };
+    const THEME = { eui: { euiColorHighlight: 'euiColorHighlight' }, darkMode: false } as EuiTheme;
```

Comment:
while type casting isn't ideal here, I couldn't find any mocks that could be used...